tests/kafkatest/services/security/security_config.py

```diff
@@ -169,7 +179,14 @@ def setup_sasl(self, node):
         jaas_conf = self.render(jaas_conf_file,  node=node, is_ibm_jdk=is_ibm_jdk,
                                 client_sasl_mechanism=self.client_sasl_mechanism,
                                 enabled_sasl_mechanisms=self.enabled_sasl_mechanisms)
-        node.account.create_file(SecurityConfig.JAAS_CONF_PATH, jaas_conf)
+        if self.static_jaas_conf:
+            node.account.create_file(SecurityConfig.JAAS_CONF_PATH, jaas_conf)
+        else:
+            match = re.search("KafkaClient *{([^}]*)}", jaas_conf.replace("\n", " "))
```

Comment:
@ewencp  Thank you for the review. Agree and that is how I wrote the code initially, but then found that the license comments got in the way. Hence the hack to do regex search and replace. Happy to change it if there is a better way to do this. Basically want to get the `KafkaClient` section without any comments before it. I was assuming  license needs to be at the start of the file before any conditionals, so I would need pattern matching to remove them from the template.